web/Controllers/LandlordController.cs

```diff
@@ -191,7 +191,15 @@ public IActionResult ViewProperties(int? id = null)
 
         var databaseResult = _propertyService.GetPropertiesByLandlord(id.Value);
         var listOfProperties = databaseResult.Select(PropertyViewModel.FromDbModel).ToList();
-        return View("Properties", new PropertiesDashboardViewModel(listOfProperties));
+        var viewModel = new PropertiesDashboardViewModel(listOfProperties);
+        var landlord = await _landlordService.GetLandlordIfExistsFromId(id);
+
+        if (landlord != null)

```

Comment:
This should never be null right? You could add another method to LandlordService `GetLandlordFromId` that throws an exception if the landlord doesn't exist. Can then make it non-null on the view model.

Reply:
Yes, you're correct! For some reason I was thinking of the situation where an Admin who isn't a landlord tried to visit the /me/ routing. But that would be caught by the nested null check at the start of the function. 

I did consider removing the nullability from the view model and then cast what we had to be non-null to fit, but the method is more reusable.